src/__tests__/create-glamorous.with-component.js

```diff
@@ -38,9 +38,13 @@ test('forwardProps can be overridden for the new component', () => {
 })
 
 test('forwards options', () => {
-  const Text = glamorous.span({color: 'red', fontSize: 20})
+  const propsToApply = {id: 'my-span', faded: true}
+  const Text = glamorous
+    .span({color: 'red', fontSize: 20})
+    .withProps(propsToApply)
   const View = Text.withComponent('div', {displayName: 'View'})
```

Comment:
Here, extending the old component with the props and styles doesn't generate two classes right ?